tests/TestSupport/Form/DynamicFieldsForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Tests\TestSupport\Form;
+
+use Yiisoft\Form\FormModel;
+use Yiisoft\Arrays\ArrayHelper;
+
+final class DynamicFieldsForm extends FormModel
+{
+    private array $fields = [];
```

Comment:
In my opinion, this creates a certain confusion between attributes / fields.

Reply:
How it was resolved? I don's see neither a comment nor a fix. @Gerych1984 @terabytesoftw